### ISSUE TITLE
When dotnet.exe fails to launch keep the console window up

### ISSUE THREAD

**Issue by davidfowl** [dotnet/roslyn-project-system#1610]
When `dotnet.exe` fails to launch it VS it currently flashes (quickly dies). This might be because the version being targeted isn't installed but I can't even tell if that's the case because it dies too fast.

**Comment by BillHiebert** [comment]
Use Ctrl+F5. The command shell should remain open allowing you to see the error.

**Comment by davidfowl** [comment]
Doesn't work. I get this error:

https://cloud.githubusercontent.com/assets/95136/23281665/14327138-f9d3-11e6-9545-c31e857c3728.png

Then when I disable launch browser. The console flashes. I *never* stays up.

I've only tried this in a web project, haven't tried a pure console app (but I can).

**Comment by BillHiebert** [comment]
Are you running IIS Express? If so, change to Kestrel and use Ctrl-F5. That's the troubleshooting I always use for these types of failures.

**Comment by davidfowl** [comment]
No, I am running self hosted. IIS Express at least gives me a nice useless error page.

**Comment by davkean** [comment]
Yeah, we should give a nice error here.
